fahclient/values.yaml

```diff
@@ -59,14 +59,16 @@ serviceAccount:
   name:
 
 securityContext:
```

Comment:
Thanks for fixing this. The only request I have left is if we could follow the Helm pattern of naming this one `podSecurityContext` and rename `containerSecurityContext` to just `securityContext`.

I noticed I had to update my values override file because it was assumed that securityContext is for the containers.

Other than this, your changes worked just fine for me. You were right about the mkdir and not needing chown.

Reply:
Yeah, I understand why to rename the variable. Good point.
will do this tonight

Reply:
And I see now that I broke the env var with my variable change too

````
        env:
          - name: PUID
            value: {{ .Values.securityContext.runAsUser | quote }}
          - name: PGID
            value: {{ .Values.securityContext.runAsGroup | quote }}
````

Reply:
I've renamed it `podSecurityContext` as suggested, and fixed the `env` block to pull from this variable name as well.